.circleci/docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,69 @@
+# vim:syntax=dockerfile
+#------------------------------------------------------------------------------
+# Dockerfile for building and testing Solidity Compiler on CI
+#
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016-2019 solidity contributors.
+#------------------------------------------------------------------------------
+FROM buildpack-deps:trusty
```

Comment:
What are the differences between the versions? Can we somehow generate these docker files?

Reply:
This is a file I just didn't want to delete "just yet". I first started with this one and wanted to use that for *any* Ubuntu version, but found out, that I had various different fixes for different ubuntu versions. That's why I started the other files with the version number suffixed, but didn't want to delete this one just yet.

Reply:
Then please keep it in a different branch.